x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.tsx

```diff
@@ -154,7 +151,7 @@ export const getAboutStepsData = (rule: Rule, detailsView: boolean): AboutStepRu
     note: note!,
     references,
     severity: {
-      value: severity,
+      value: severity as SeverityValue,
```

Comment:
As above, would it be possible to start using the common types here?

e.g.

https://github.com/spong/kibana/blob/d2156b7285c0397fe93a12cd8f1464ac430bc0e8/x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts#L256-L257